vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -50,7 +50,7 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
-      new_result.merge(
+      new_result.merge!(
```

Comment:
Fair enough for me.  What you have now fits in with the style in the rest of the file.